### PR TITLE
[TPI-5182] Round Up Invoice Amount

### PR DESCRIPTION
**Pull request by kevinalfianto**
## Description

- round up invoice amount

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

image

invoice:

image

multi shipping:
image

invoice:
image

### CODE REVIEW

File: Xendit/M2Invoice/Controller/Checkout/Invoice.php [hunk 1]
```diff
@@ -95,11 +95,12 @@ private function getApiRequestData($order)
             $items[] = (object) $item;
         }
 
+        $amount = $order->getBaseCurrencyCode() == 'IDR' ? ceil($order->getTotalDue()): $order->getTotalDue();
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Controller/Checkout/Invoice.php:98]: you forgot the multishipping one

Comment (kevinalfianto) [reply Xendit/M2Invoice/Controller/Checkout/Invoice.php:98]: updated here, 
063713a and  b0d00ac

**Review by IreneGohtami — APPROVED** [review]
Can you also check individual payment method whether there is an (int) casting? If so, please remove them as well. I

**Comment by kevinalfianto** [comment]
you mean the invoice without multi shipping? Normal invoice don't cast the (int) 
you can see here https://github.com/xendit/xendit-magento-payment-module/blob/master/Xendit/M2Invoice/Controller/Checkout/Invoice.php#L102

image

**Comment by IreneGohtami** [comment]
> you mean the invoice without multi shipping? Normal invoice don't cast the (int) you can see here https://github.com/xendit/xendit-magento-payment-module/blob/master/Xendit/M2Invoice/Controller/Checkout/Invoice.php#L102
> 
> image

I see an (int) casting in ccmultishipping & ccsubscription, we can change this to ceil as well
And there's a bunch of ceil(...) in `function isAvailable`, should we standardize this to check for IDR as well?

**Comment by kevinalfianto** [comment]
updated for cc multishipping here 87d3ccd
for other payment that has isAvailable function already have checker only for IDR so no need to add IDR checker.
There are 2 method that don't have it though, cc and ccsubscription. I think to leave it like that at the moment

**Review by yanoandri — APPROVED** [review]

**Pull request merged**